src/Service/InstallManager.php

```diff
@@ -40,15 +40,47 @@ public function install()
     {
         $state = true;
         $state &= $this->createCarrier();
+        $state &= $this->createExportTable();
 
         $this->installDefaultConfiguration();
 
         return $state;
     }
 
+    /**
+     * @return bool
+     */
+    public function createExportTable()
+    {
+        $query = "
+            CREATE TABLE IF NOT EXISTS ps_oyst_exported_catalog
+            (
+                productId INT,
+                productAttributeId INT,
+                importId VARCHAR(60),
+                hasBeenExported TINYINT DEFAULT 0
+            );  
+        ";
+
+        return $this->db->execute($this->prefixQuery($query));
+    }
+
+    /**
+     * @return bool
+     */
+    public function dropExportTable()
+    {
+        $query = "
+            DROP TABLE IF EXISTS ps_oyst_exported_catalog;
+        ";
+
+        return $this->db->execute($this->prefixQuery($query));
+    }
+
     public function uninstall()
     {
         $this->removeCarrier();
+        $this->dropExportTable();
```

Comment:
Are we sure we want to delete this table ?
At first the module deleted tables it had created on uninstallation. But we asked me to not delete this table in case of bad manipulation by the merchant.